usaspending_api/etl/management/commands/load_usaspending_contracts.py

```diff
@@ -84,7 +84,6 @@ def handle(self, *args, **options):
                 "gfe_gfp": h.up2colon(row['gfe_gfp']),
                 "cost_or_pricing_data": h.up2colon(row['costorpricingdata']),
                 "type_of_contract_pricing": h.up2colon(row['typeofcontractpricing']),
-                "type_of_contract_pricing_description": CONTRACT_PRICING_TYPES_D.get(h.up2colon(row['typeofcontractpricing'])),
```

Comment:
I like how pulling all this stuff out into update_model_description_fields makes this function shorter and less complex.